### PR TITLE
Announced prop added to ic-status-tag

### PR DESCRIPTION
**Pull request by gd2910**
The status tag had a role property that was not being read out by assistive technology.

## Summary of the changes
Added an 'Announced' prop that conditionally renders the status-tag component as an aria-live region. This is a fix for the aria-label being read out incorrectly. This is set as 'false' by default. 

When set as 'true' the status-tag will act as an aria-live region.

## Related issue
Connected ticket: https://github.com/mi6/ic-ui-kit/issues/20

This is an updated PR from a now deleted previous PR/branch

## Checklist
- [x] I have added relevant unit and visual regression tests.
- [x] I have manually accessibility tested any changes, if relevant.
- [ ] I have ensured any changes match the Figma component library.

### CODE REVIEW

File: packages/web-components/src/components/ic-status-tag/ic-status-tag.tsx [hunk 1]
```diff
@@ -27,6 +27,10 @@ export class StatusTag {
    * If `true`, the small styling will be applied to the status tag.
    */
   @Prop() small?: boolean = false;
+  /**
+   * If 'true', the component will act as an 'aria-live' region.
```
Comment (jd239) [inline packages/web-components/src/components/ic-status-tag/ic-status-tag.tsx:31]: Is it worth mentioning it adds the `role=status` to the component as I was initially confused why it talks about aria-live?

File: packages/web-components/src/components/ic-status-tag/ic-status-tag.tsx [hunk 2]
```diff
@@ -36,9 +40,12 @@ export class StatusTag {
   }
 
   render() {
-    const { label, status, appearance, small } = this;
+    const { label, status, appearance, small, announced } = this;
     return (
-      <Host role="status">
+      <Host
+        role={announced ? "status" : null}
+        aria-label={"Status. " + `${label}`}
```
Comment (jd239) [inline packages/web-components/src/components/ic-status-tag/ic-status-tag.tsx:47]: This can be done as
```jsx
aria-label={`Status. ${label}`}
```

**Review by jd239 — CHANGES_REQUESTED** [review]
Two minor comments. 👍

**Review by GCHQ-Developer-847 — CHANGES_REQUESTED** [review]
Just noticed a couple of tiny things

File: packages/web-components/src/components/ic-status-tag/readme.md [hunk 3]
```diff
@@ -9,6 +9,7 @@
 
 | Property             | Attribute    | Description                                                     | Type                                              | Default     |
 | -------------------- | ------------ | --------------------------------------------------------------- | ------------------------------------------------- | ----------- |
+| `announced`          | `announced`  | If 'true', the component will act as an 'aria-live' region.     | `boolean`                                         | `false`     |
```
Comment (jd239) [inline packages/web-components/src/components/ic-status-tag/readme.md:12]: Think you need to run the build again to update this readme file. `npm run build`

**Review by jd239 — APPROVED** [review]

**Review by GCHQ-Developer-847 — APPROVED** [review]

**Review by jd239 — APPROVED** [review]

**Pull request merged**